### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by batoolfatima2135**
In this branch, I have :

- Deleted all the boilerplate from Create React App (text, images, styles).
- Created a new directory called Components.
- Inside components, created a new Calculator.js file.
- In Calculator.js, created a React component that matches this design:
 
image
 
- Implemented the design only (JSX and CSS)
- Used functional components
- Created 3 child component
- Imported the calculator and used it in the top-level component: App.js
- Tested it in the browser; it is rendered without problems.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      <div className="col-5 mt-5">
+        <ResultScreen />
+        <RowCalculator items={['AC', '+/-', '%', '\u00F7', '7', '8', '9', '\u00D7', '4', '5', '6', '-', '1', '2', '3', '+']} />
+        <EndRow items={['0', '.', '=']} />
+
+      </div>
+    </div>
+  );
+}
+export default Calculator;
```
Comment (nahnahdev) [inline src/components/calculator.js]: - [ OPTIONAL ] To adhere to best practices, you can create separate files for each component, exporting them individually. For example, you can have a file for `RowCalculator`, `ResultScreen`, `EndRow`, and `Calculator`. Each file should contain the code for its respective component. :100: 

- [ ] By organizing your code this way, you can improve code maintainability, reusability, and readability. :100:

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (nahnahdev) [inline src/components/calculator.js]: [ OPTIONAL ] I recommend avoiding the use of `eslint disable` commands in your projects, as these errors and warnings are typically helpful for identifying areas where you can improve your code. By resolving these issues, you can write more clean and readable code that adheres to industry standards, which can make it easier to maintain your codebase over time. :100:

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @batoolfatima2135 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights :fire: 
- LInter well setup and passing :heavy_check_mark: 
- Professional README :heavy_check_mark: 
- Calculator looks as expected :heavy_check_mark: 
- Child components incorporated into the `Calculator` component :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 1 above)
Comment (batoolfatima2135) [reply src/components/calculator.js]: @NahnahAJ that's a very good piece of advice. Implementing it right away! 👍 
Thank you 😊

File: src/components/calculator.js (hunk 2 above)
Comment (batoolfatima2135) [reply src/components/calculator.js]: @NahnahAJ Actually package was in the dependencies of my project but still, the linter kept showing errors that the package is not in the dependencies that's why I had to do it
Here is the screenshot of dependencies in `pakage.json`. 
Capture
And here is the error:
Screenshot (2)

Let me know if you have a solution to this. 👍

**Pull request merged**